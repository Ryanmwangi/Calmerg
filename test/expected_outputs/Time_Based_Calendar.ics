BEGIN:VCALENDAR
PRODID:-//Your Product ID//EN
VERSION:2.0
NAME:Time Based Calendar
BEGIN:VEVENT
UID:20231109T110000-001@example.com
SUMMARY:team_meeting_calendar Team Meeting
DTSTART:20231109T140000
DTEND:20231109T150000
END:VEVENT
BEGIN:VEVENT
UID:20231108T090000-001@example.com
SUMMARY:work_task Work Task
DTSTART:20231108T120000
DTEND:20231108T130000
END:VEVENT
END:VCALENDAR