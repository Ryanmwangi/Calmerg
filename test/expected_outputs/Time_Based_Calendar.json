{
  "linkGroupName": "Time Based Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\team_meeting_calendar.ics",
      "prefix": "team_meeting_calendar",
      "override": false
    },
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\work_task_calendar.ics",
      "prefix": "work_task",
      "override": false
    }
  ]
}